lib/bolt/util.rb

```diff
@@ -77,6 +77,15 @@ def read_optional_yaml_hash(path, file_name)
         File.exist?(path) ? read_yaml_hash(path, file_name) : {}
       end
 
+      def first_runs_free
+        path = Bolt::Config.user_path || Bolt::Config.system_path
```

Comment:
I'm not too familiar with the cases where the running user does not have a homedir, but are they guaranteed to have permissions to write to the system path?

Reply:
Probably not.

Reply:
Ok, well, I think if the user doesn't have a homedir it's also most likely a case where we don't need to print a welcome message anyway. Rather than trying to catch errors and write the file somewhere else I just made `Bolt::Config.user_path` succeeding a requirement for printing the text to begin with.